src/main/scala/de/m7w3/signal/ApplicationContext.scala

```diff
@@ -1,20 +1,24 @@
 package de.m7w3.signal
 
+import de.m7w3.signal.account.AccountHelper
+import de.m7w3.signal.events.SignalDesktopEventDispatcher
 import de.m7w3.signal.store.model.Schema
 import de.m7w3.signal.store.{DBActionRunner, DatabaseLoader, SignalDesktopApplicationStore, SignalDesktopProtocolStore}
+import monix.execution.atomic.Atomic
 import org.apache.logging.log4j.Level
 import org.apache.logging.log4j.core.config.Configurator
-import org.slf4j.LoggerFactory
 import slick.driver.H2Driver.api._
 
 import scala.util.Try
 
-trait ApplicationContext
+trait ApplicationContext {
+  def close(): Unit
+}
 
 
-case class ContextBuilder(config: Config.SignalDesktopConfig) extends ApplicationContext{
+case class ContextBuilder(config: Config.SignalDesktopConfig,
```

Comment:
It only takes a reference that will be filled once the context is built.
Sounds strange, now that i write about it.
will change.